src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -371,6 +371,9 @@ def acs_get_credentials(dns_prefix, location):
     home = os.path.expanduser('~')
```

Comment:
Good call, I've updated it to be `k8s_...` instead of `acs_...` now, in alignment with `k8s_install_cli` etc.